packages/suite/src/components/suite/AmountUnitSwitchWrapper.tsx

```diff
@@ -47,7 +47,10 @@ export const AmountUnitSwitchWrapper = ({ symbol, children }: AmountUnitSwitchWr
             content={<Translation id={areSatsDisplayed ? 'TR_TO_BTC' : 'TR_TO_SATOSHIS'} />}
         >
             <Container
-                onClick={toggleBitcoinAmountUnits}
+                onClick={e => {
```

Comment:
Nitpick: would be better to have it in separate function such as `handleToggleBitcoinAmountUnits` for better readability

Reply:
:heavy_check_mark: